rss-client/src/app/Quiz/employee/quiz-page/quiz-page.component.scss

```diff
@@ -0,0 +1,65 @@
+// .quiz {
+//     box-shadow: 0 0 10px 5px rgba(0,0,0,0.25);
+//     border-radius: 5px;
+//     display: flex;
+//     margin: 2rem auto;
+//     width: 67%;
+// }
+
+// .quizDetails {
+//     width: 100%;
+//     padding: 1rem;
+// }
+
+// .quiz img {
+//     width: 350px;
+//     height: auto;
+//     display: block;
+//     object-fit: cover;
+// }
+
```

Comment:
Why Comment?

Reply:
This is ui/ux code. us1 would like to keep this commented out code in case we want to go back to old styling.